vignettes/Workflow.Rmd

```diff
@@ -0,0 +1,479 @@
+---
+title: "Some Basic Analysis of ChIP-Seq Data"
+authors:
+  - name: "Deepayan Sarkar"
+  - name: "Robert Gentleman"
+  - name: "Michael Lawrence"
+  - name: "Zizhen Yao"
+  - name: "Oluwabukola Bamigbade" 
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date:  "`r format(Sys.time(), '%B %d , %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Some Basic Analysis of ChIP-Seq Data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteKeywords{chip-seq, sequencing, visualization}
+  %\VignettePackage{chipseq}
+  
+output:
+  BiocStyle::html_document
+---
+
+Our goal is to describe the use of Bioconductor software to perform some
+basic tasks in the analysis of ChIP-Seq data. We will use several
+functions in the as-yet-unreleased [chipseq]{.sans-serif} package, which
+provides convenient interfaces to other powerful packages such as
+[ShortRead]{.sans-serif} and [IRanges]{.sans-serif}. We will also use
+the [lattice]{.sans-serif} and [rtracklayer]{.sans-serif} packages for
+visualization.
+
+```{r setup,echo=TRUE, results="hide"}
+library(chipseq) 
+library(GenomicFeatures) 
+library(lattice)
+```
+
+# Example data 
+
+The `cstest` data set is included in the [chipseq]{.sans-serif} package
+to help demonstrate its capabilities. The dataset contains data for
+three chromosomes from Solexa lanes, one from a CTCF mouse ChIP-Seq, and
+one from a GFP mouse ChIP-Seq. The raw reads were aligned to the
+reference genome (mouse in this case) using an external program (MAQ),
+and the results read in using the the `readAligned` function in the
+[ShortRead]{.sans-serif}, in conjunction with a filter produced by the
+`chipseqFilter` function. This step filtered the reads to remove
+duplicates, to restrict mappings to the canonical, autosomal chromosomes
+and ensure that only a single read maps to a given position. A quality
+score cutoff was also applied. The remaining data were reduced to a set
+of aligned intervals (including orientation). This saves a great deal of
+memory, as the sequences, which are unnecessary, are discarded. Finally,
+we subset the data for chr10 to chr12, simply for convenience in this
+vignette.
+
+We outline this process with this unevaluated code block:
+```{r preprocess, eval=FALSE}
```

Comment:
Add a blank line before line 55. We also have a blank line before and after code blocks.